packages/react/src/editor/BlockNoteView.tsx

```diff
@@ -2,21 +2,23 @@ import {
   BlockNoteEditor,
   BlockSchema,
   InlineContentSchema,
-  StyleSchema,
   mergeCSSClasses,
+  StyleSchema,
 } from "@blocknote/core";
-import { MantineProvider, createStyles } from "@mantine/core";
+import { MantineProvider } from "@mantine/core";
 import { EditorContent } from "@tiptap/react";
-import { HTMLAttributes, ReactNode, useMemo } from "react";
+import { HTMLAttributes, ReactNode, useEffect } from "react";
 import usePrefersColorScheme from "use-prefers-color-scheme";
-import { Theme, blockNoteToMantineTheme } from "./BlockNoteTheme";
+import { Theme, themeToCSSVariables } from "./BlockNoteTheme";
 import { FormattingToolbarPositioner } from "../components/FormattingToolbar/FormattingToolbarPositioner";
 import { HyperlinkToolbarPositioner } from "../components/HyperlinkToolbar/HyperlinkToolbarPositioner";
 import { ImageToolbarPositioner } from "../components/ImageToolbar/ImageToolbarPositioner";
 import { SideMenuPositioner } from "../components/SideMenu/SideMenuPositioner";
 import { SlashMenuPositioner } from "../components/SlashMenu/SlashMenuPositioner";
 import { TableHandlesPositioner } from "../components/TableHandles/TableHandlePositioner";
 import { darkDefaultTheme, lightDefaultTheme } from "./defaultThemes";
+import "@mantine/core/styles.css";
```

Comment:
remove css imports here

(similar to https://github.com/TypeCellOS/BlockNote/pull/467/files)